packages/components/pie-card-container/test/component/pie-card-container.spec.ts

```diff
@@ -229,4 +229,47 @@ test.describe('PieCardContainer - Component tests', () => {
             });
         });
     });
+
+    test.describe('Prop: `padding` & `PaddingX`', () => {
```

Comment:
Guessing this can just be:

```suggestion
    test.describe('Prop: `padding`', () => {
```